app/models/volunteer_op.rb

```diff
@@ -34,4 +42,16 @@ def self.contains_description?(text)
   def self.contains_title?(text)
     arel_table[:title].matches(text)
   end
+  
+  def address_complete?
+    address.present? && postcode.present?
+  end 
+  
+  private
+  
+  def clear_lat_lng
+    return if address_complete?
+    self.longitude = nil
+    self.latitude = nil
+  end
```

Comment:
@tansaku the `clear_lat_lng` method will affect the imported volunteer op from do-it because we are persisting only coordinates ( without address). 
There is an open PR that fix this issue https://github.com/AgileVentures/LocalSupport/pull/335 but it make sense now to fix this issue before merging the PR.